widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForDialogs.kt

```diff
@@ -375,12 +381,33 @@ class ActivityWatcherForDialogs(
 
         alertDialog = Dialogs.showUpgradeDialog(contextWithStyle, theme, mediaUpgrade, {
             dialogController.dismissCurrentDialog()
-            // TODO: 07.02.2023 handle media request accepting in the scope of respective ticket
+            mediaUpgrade.mediaUpgradeOffer.accept {
+                if (it == null) {
```

Comment:
NIT: Think it would nice if you added a name to the argument
```suggestion
            mediaUpgrade.mediaUpgradeOffer.accept { error ->
                if (error == null) {
```